bigtable/tests/instance_admin_emulator.cc

```diff
@@ -86,7 +86,41 @@ class InstanceAdminEmulator final
       grpc::ServerContext* context,
       btadmin::PartialUpdateInstanceRequest const* request,
       google::longrunning::Operation* response) override {
-    return grpc::Status(grpc::StatusCode::UNIMPLEMENTED, "not implemented");
+    std::string name = request->instance().name();
+    auto it = instances_.find(name);
+    if (it == instances_.end()) {
+      return grpc::Status(grpc::StatusCode::NOT_FOUND, "instance not found");
+    }
+
+    auto stored_instance = it->second;
```

Comment:
I think this should be `auto& stored_instance` or the changes below are lost after the function returns, right?